### ISSUE TITLE
RapidUsertests

### ISSUE THREAD

**Issue by weliveindetail** [ImmunHelden/ImmunHelden.de#144]
Ergebnis des Calls mit Benedikt von RapidUsertests: Sie unterstützen uns jetzt direkt und dann nochmal im späteren Verlauf, wenn wir z.B. erste Partnerorgas dabei haben!

Testkonzept: https://slack-files.com/T011CE9BQS2-F0152TML2CX-88328f3a3a

**Comment by ImmunHeldin-CE** [comment]
Link zu den Tests und den User-Videos: 
https://rapidusertests.com/customer/dashboard/overview/8872
Logindaten:
christine@immunhelden.de 
Passwort wie bekannt :)

**Comment by ImmunHeldin-CE** [comment]
Passwort wie immer :)

**Comment by ImmunHeldin-CE** [comment]
Alle Videos sind da, die Auswertung steht noch aus. Ich habe Bene kontaktiert für ein kurzes methodisches Onboarding.

**Comment by weliveindetail** [comment]
Gibt es hier noch was zu tun?

**Comment by ImmunHeldin-CE** [comment]
> Gibt es hier noch was zu tun?

Ja, das wichtigste, die Auswertung. Das hat allerdings wenig mit unserer Partnerschaft mit Userlutions zu tun :)

**Comment by vikiarn** [comment]
Hier die Auswertung: https://docs.google.com/document/d/1Q15PkQBkwx7zpGgaU--YwxpWg4q8-S-enmsX34TAfVY/edit

**Comment by vikiarn** [comment]
Bearbeitung der Verbesserungsvorschläge: https://docs.google.com/document/d/1uyMSVUm7TGtyBs8aJkCN6PetcpSu2p2XeeeRiMyYe_E/edit
